apps/ybribe/components/rewardFeed/RewardFeedTableRow.tsx

```diff
@@ -52,6 +52,10 @@ function	RewardFeedTableRow({currentRewardAdded}: {currentRewardAdded: TYDaemonG
 
 	const	gaugeItem = gaugesObject[toAddress(currentRewardAdded.gauge)];
 
+	if (!gaugeItem) {
+		return null;
+	}
```

Comment:
Sounds good, done with https://github.com/yearn/yearn.fi/pull/161/commits/84dbbb380f1a2ea3a45db75a51bd7ed9d18bfe0d

Should we add loading states too? We're not doing that a whole lot but it could help, having some skeleton loading

Reply:
Let's do it in a separate PR